crates/oxc_transformer/src/decorator/legacy.rs

```diff
@@ -0,0 +1,284 @@
+//! Typescript Experimental Decorator
```

Comment:
`Experimental` has become confusing, let's not mention it except use the term legacy decorator (`--experimentalDecorators`) explicitly.